docker/rocal-with-pytorch.dockerfile

```diff
@@ -52,7 +42,10 @@ RUN DEBIAN_FRONTEND=noninteractive apt-get -y install python3 python3-pip git g+
 
 # install MIVisionX 
 RUN git clone https://github.com/ROCm/MIVisionX.git && cd MIVisionX && \
-        mkdir build && cd build && cmake -DBACKEND=HIP ../ && make -j8 && make install && cd 
+        mkdir build && cd build && cmake -DBACKEND=HIP ../ && make -j8 && make install && cd
```

Comment:
can we get mivisionx from package too?

Reply:
right now the mivisionx package has no vx_rpp